pkgs/development/python-modules/mizani/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, matplotlib
+, palettable
+, pandas
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "mizani";
+  version = "0.7.3";
+
+  src = fetchFromGitHub {
+    owner = "has2k1";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "04r53dp5jbklv8l9ncgc5wiq0gx25y73h65gmmbbfkxwgsl3w78l";
+  };
+
+  # Patch out the pytest coverage checking.
+  patchPhase = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
Is there a reason to use postPath instead of patchPhase?

Reply:
`patchPhase` will override the patch phase. The `pre*` and `post*` phases are addition and execute before or after the default phases.

Reply:
Fair enough... since I didn't have any patches I figured it would be ok to just take over patchPhase but I can see why that may not be desirable in general.